src/pages/settings/Security/SecuritySettingsPage.js

```diff
@@ -9,13 +11,40 @@ import * as Expensicons from '../../../components/Icon/Expensicons';
 import ScreenWrapper from '../../../components/ScreenWrapper';
 import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
 import MenuItem from '../../../components/MenuItem';
+import compose from '../../../libs/compose';
+import ONYXKEYS from '../../../ONYXKEYS';
+import * as Session from '../../../libs/actions/Session';
 
 const propTypes = {
+    /* Onyx Props */
+
+    /** Holds information about the users account that is logging in */
+    account: PropTypes.shape({
+        /** Whether this account has 2-FA enabled or not */
+        requiresTwoFactorAuth: PropTypes.bool,
+    }),
+
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    account: {},
+};
+
 const SecuritySettingsPage = (props) => {
```

Comment:
I would change this to be a [function instead of arrow,](https://expensify.slack.com/archives/C01GTK53T8Q/p1682443591606479) given that we are moving to that direction